src/ovs/aca_ovs_control.cpp

```diff
@@ -85,7 +93,35 @@ int ACA_OVS_Control::control()
 
 void ACA_OVS_Control::dump_flows(const char *bridge, const char *opt)
 {
-  OVS_Control::get_instance().dump_flows(bridge, opt);
+  bool show_stats = true;
+  OVS_Control::get_instance().dump_flows(bridge, opt, show_stats);
+}
+
+int ACA_OVS_Control::flow_exits(const char *bridge, const char *flow)
+{
+  int rc = -EINVAL;
+  bool show_stats = false;
+  
+  rc = OVS_Control::get_instance().dump_flows(bridge, flow, show_stats);
+
+  return rc;
+}
+
+void ACA_OVS_Control::add_flow(const char *bridge, const char *opt)
```

Comment:
I was hoping this add_flow function would accept a list of parameter, like table number (default=0) priority, etc. And it could return a meaningful error for failure instead of returning void. Is that possible?

Reply:
are we able to get meaningful error information if OVS_Control::get_instance().add_flow(bridge, opt) failed?